ericus123/PORTFOLIO_UI
# Fix design issues

#### What does this PR do?
Fix design issues, responsiveness and so on..

#### Description of Task to be completed?

- Check if all pages are responsive
- Improve designs


#### How should this be manually tested?

How should this be manually tested?
- Clone the repo `git clone https://github.com/aericus123/PORTFOLIO_UI.git`
- Switch your working directory `cd PORTFOLIO_UI`
- Checkout the branch `git checkout fix-design-issues`
- Create a` .env file` in the root folder and update it using data found in `env-example`
- Install all dependencies `yarn install`
- Start the server `yarn run dev`
- Visit login the app on localhost url and test it 
- You should see that all pages are responsive and the designs are good

#### Any background context you want to provide?
N/A
#### What are the relevant trello/pivotal tracker stories?
N/A
#### Screenshots (if appropriate)

#### Questions:

N/A
